src/index.js

```diff
@@ -7,17 +7,22 @@ export class PhoenixLiveViewDropzone {
     var node = document.createElement("div");
     this.el.appendChild(node);
 
-    var liveView = this;
```

Comment:
So the reason for doing, er, this, was because `this` inside the callbacks later on was not the hook anymore, e.g. when called from Elm it was not the liveview `this` anymore. So I'm pretty certain it needs to be captured here, and it should be the current rendered elements hook context at this point, which shouldn't change, so I don't think you need to change this unless I'm not understanding the issue?

Reply:
You are right, I got to the code's current state via more error than trial, and the way my code ended up is less straightforward than how you had done it originally.

Reasoning about `this` with LiveView callbacks has certainly been an experience! I believe I've got my mind wrapped around which framework sets `this` to what context now, I'll update the PR when I have a chance.

Thanks!